pkg/scheduler/preemption/preemption_test.go

```diff
@@ -684,6 +693,44 @@ func TestPreemption(t *testing.T) {
 			},
 			wantPreempted: sets.New("/low-alpha", "/low-beta"),
 		},
+		"preempt newer workloads with the same priority": {
+			admitted: []kueue.Workload{
+				*utiltesting.MakeWorkload("wl1", "").
+					Priority(2).
+					Request(corev1.ResourceCPU, "3").
+					Admit(utiltesting.MakeAdmission("preventStarvation").Assignment(corev1.ResourceCPU, "default", "3").Obj()).
+					Obj(),
+				*utiltesting.MakeWorkload("wl2", "").
+					Priority(1).
+					Creation(time.Now()).
+					Request(corev1.ResourceCPU, "2").
+					Admit(utiltesting.MakeAdmission("preventStarvation").Assignment(corev1.ResourceCPU, "default", "2").Obj()).
+					Obj(),
```

Comment:
Add another workload with the same priority that has a time older than 15 seconds

Reply:
in order to have 2 workloads with the same priority where one is preempted (the newer one), but not the other.

Reply:
done